pkg/collector/corechecks/embed/jmx/fixtures/hazelcast.yaml

```diff
@@ -0,0 +1,89 @@
+instances:
+  - host: localhost
+    port: 1099
+    is_jmx: true
+
```

Comment:
I think the test would be more valuable if there was an additional instance here that's not a JMXFetch config, what do you think?